sdk/export/metric/metric.go

```diff
@@ -114,30 +319,48 @@ func NewDescriptor(
 	}
 }
 
+// Name returns the metric instrument's name.
 func (d *Descriptor) Name() string {
 	return d.name
 }
 
-func (d *Descriptor) MetricKind() Kind {
+// MetricKind returns the kind of instrument: counter, gauge, or
+// measure.
+func (d *Descriptor) MetricKind() MetricKind {
 	return d.metricKind
 }
 
+// Keys returns the recommended keys included in the metric
+// definition.  These keys may be used by a Batcher as a default set
+// of grouping keys for the metric instrument.
 func (d *Descriptor) Keys() []core.Key {
 	return d.keys
 }
 
+// Description provides a human-readable description of the metric
+// instrument.
 func (d *Descriptor) Description() string {
 	return d.description
 }
 
+// Unit describes the units of the metric instrument.  Unitless
+// metrics return the empty string.
 func (d *Descriptor) Unit() unit.Unit {
 	return d.unit
 }
 
+// NumberKind returns whether this instrument is declared over int64
+// or a float64 values.
 func (d *Descriptor) NumberKind() core.NumberKind {
 	return d.numberKind
 }
 
+// Alternate returns true when the non-default behavior of the
```

Comment:
`Alternate` as a naming is very very confusing, would suggest a more intuitive name (but I'm struggling to figure out what that might be, as you point out, there are 3 different ways that we might vary from the instrument's default behavior.)